BraveRewardsUI/Settings/Auto-Contribute/Details/AutoContributeCell.swift

```diff
@@ -48,6 +48,8 @@ class AutoContributeCell: UITableViewCell, TableViewReusable {
     siteStackView.addArrangedSubview(siteImageView)
     siteStackView.addArrangedSubview(siteNameLabel)
     
+    siteImageView.snp.removeConstraints()
```

Comment:
What's this for?

Reply:
siteImageView's constructor takes a height which it uses to set a height constraint. But the imageview is placed inside a stackview and the stackview is inside a self sizing UITableviewcell. When it's drawn because of conflicting constraints, causes jerks. Removing this makes imageview draw relative to the stackview.

Reply:
Ahhh okay cool didn't notice it was a custom image view